duck:
  hh: 21
  hl: 0
  sh: 255
  sl: 220
  vh: 255
  vl: 200
lane_image:
  bottom_left_x: 720
  bottom_left_y: 444
  bottom_right_x: 0
  bottom_right_y: 440
  top_left_x: 262
  top_left_y: 160
  top_right_x: 419
  top_right_y: 160
white:
  hh: 255
  hl: 0
  sh: 80
  sl: 0
  vh: 255
  vl: 180
yellow:
  hh: 80
  hl: 20
  sh: 255
  sl: 80
  vh: 255
  vl: 100


